stylesheet.css

```diff
@@ -949,8 +956,46 @@ button:hover {
     padding: 14px 36px 14px 24px;
   }
 
+  .menu-icon {
+    position: fixed;
+    right: 35px;
+    z-index: 100;
+  }
+
+  #menuBox {
+    width: 0;
+    height: 0;
+    background: rgba(7, 18, 114, 0.9);
+    position: fixed;
+    top: 0;
+    right: 0;
+    border-bottom-left-radius: 100%;
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. You may consider using the some color, typo and spacing between your elements.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201862665-ba3ff9a3-244f-415b-8811-be9d7f391ea2.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201862872-b3cf87e0-10eb-4613-bc03-70363a6fac92.png)